M2/libraries/ntl/Makefile.in

```diff
@@ -2,12 +2,11 @@
 LICENSEFILES = doc/copying.txt
 # @configure_input@
 # PATCHFILE = @abs_srcdir@/patch-$(VERSION)
-VERSION = 10.5.0
+VERSION = 11.5.1
 TARDIR = ntl-$(VERSION)
 BUILDDIR = $(UNTARDIR)/$(TARDIR)/src
 TARFILE = ntl-$(VERSION).tar.gz
-# URL = http://shoup.net/ntl
-URL = http://macaulay2.com/Downloads/OtherSourceCode
+URL = http://shoup.net/ntl
```

Comment:
The website redirects to https://libntl.org/. Maybe use that instead?

Reply:
Good catch -- thanks!